FlowCrypt/Controllers/Inbox/InboxItem.swift

```diff
@@ -22,6 +22,10 @@ struct InboxItem: Equatable {
         return messages.contains(where: \.attachments.isNotEmpty)
     }
 
+    var onlyHavePublicKey: Bool {
+        return messages.count == 1 && messages.contains(where: \.hasPublicKeyAttachment)
```

Comment:
why `messages.count` should equal to 1?

for example, user sent me public key - app shows `public key attachment` icon, then I replied to him and now app shows `attachment` icon. I think it should show `public key attachment` icon for any number of messages

Reply:
> No, let's show only attachment icon in this case. So public key icon will be shown when thread contains only public key attachment

Reply:
I added that logic because of this comment. Maybe I understood something wrong?

Reply:
Yeah, I meant that we should show `public key attachment` icon only for cases when thread contains only public key attachment (or public key attachments), but if it has public key attachment and some other attachments - then show `attachment` icon, not depending on number of messages in thread.

Reply:
Ohh... I was confused because I thought I added logic to check if message contains only public key but now realized I only checked messages count. Let me fix it